README.md

```diff
@@ -36,23 +36,31 @@ See [./docs/OffsetHelper.md](./docs/OffsetHelper.md) for detailed documentation.
 
 ## Commands
 
-Use the following commands to compile, test and deploy the contracts:
+Use the following commands to compile and test the Offset Helper:
 
 ```
 yarn compile
 yarn test      # test using a polygon fork
-yarn coverage  # test using a polygon fork with coverage report
-yarn deploy
 ```
 
-Documentation can be auto-generated from the contract's [natspec](https://docs.soliditylang.org/en/latest/natspec-format.html) in [./docs/](./docs/) using
+Deploy the contract locally with:
+
+```
```

Comment:
Little tip: if you are not specifying a language after the triple backticks, then it's nicer style to skip the backticks and instead indent the code by 4 columns. It renders the same but keeps the source closer to the original vision of Markdown which was a near invisible form of markup which keeps the original source looking like beautifully formatted plaintext.  Actually even if you want to specify the language, it's arguably worth using the 4-column indent.